test/test_elixirs.jl

```diff
@@ -38,6 +50,14 @@ const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples")
     λ = eigvals(Matrix(A))
     @test maximum(real, λ) < 10*length(λ) * eps(real(eltype(λ)))
   end
+
+  @testset "Test Jacobian of DG (3D)" begin
+    A, b = Trixi.compute_linear_structure(joinpath(EXAMPLES_DIR, "3d", "parameters.toml"),
+                                          polydeg=2, initial_refinement_level=1)
+    J = Trixi.compute_jacobian_dg(joinpath(EXAMPLES_DIR, "3d", "parameters.toml"),
+                                          polydeg=2, initial_refinement_level=1)
+    @test isapprox(Matrix(A), J)
```

Comment:
Why is there no eigenvalue test like for 2D? Is it too expensive?

Reply:
It takes some CPU time to do, furthermore, there is no real added value as we already test 3D with the linear test case...

Reply:
OK.